osbenchmark/aggregator.py

```diff
@@ -214,9 +214,9 @@ def calculate_weighted_average(self, task_metrics: Dict[str, List[Any]], iterati
 
         return weighted_metrics
 
-    def calculate_rsd(self, values):
+    def calculate_rsd(self, values, metric_name: str):
```

Comment:
Nit: We should also incldue the type hints for `values` if we are providing one for `metric_name`